source/SnapDump-Server/SnapDumpAPI.class.st

```diff
@@ -0,0 +1,34 @@
+Class {
+	#name : #SnapDumpAPI,
+	#superclass : #OpenAPI,
+	#classVars : [
+		'path'
+	],
+	#category : #'SnapDump-Server-REST'
+}
+
+{ #category : #accessing }
+SnapDumpAPI class >> rootCallClass [
+	^ SDOpenAPICall 
+]
+
+{ #category : #accessing }
+SnapDumpAPI >> info [
+	^ OAInfo new 
+		description: 'SnapDump API';
+		version: '0.7';
```

Comment:
I use 0.7 here as it would be the next release version. (to have this automated i think we have a related issue and the outcome could be used here )